### DISCUSSION TITLE
Connectivity via S403 header?

### DISCUSSION THREAD

**Discussion by tiekenz** [revk/xESP32-Faikout#48] [General]
I've just received my PCB from Amazon, and although I had assumed that connecting it wouldn't be an issue as I've got a relatively recent unit, it seems that my Daikin Cora unit (FTXM50UVMZ) has no internal S21 header, but only an S403 header on the internal PCB, so from my understanding I would need an additional interface (KRP980B1) to provide an S21 port?

**Comment by revk** [comment]
I have no idea. But so let me know how it goes.

**Comment by tiekenz** [comment]
I ended up getting the Daikin daughterboard BRP980B42 which provides an S21 header and normally acts as an adapter to a BRP072C42 wifi unit. Unfortunately, although I can successfully connect and configure the Faikin side of things (and have updated to the latest firmware), the module doesn't receive any information from the heatpump, simply saying "System is off line."

http://192.168.0.56/aircon/get_control_info:
ret=OK,pow=0,mode=6,adv=,stemp=nan,shum=0,dt1=nan,dt2=nan,dt3=nan,dt4=nan,dt5=nan,dt7=nan,dh1=0,dh2=0,dh3=0,dh4=0,dh5=0,dh7=0dhh=0,b_mode=6,b_stemp=nan,b_shum=0,alert=255,f_rate=A,f_dir=0,dfr1=0,dfr2=0,dfr3=0,dfr4=0,dfr5=0,dfr7=0,dfrh=0,dfd1=0,dfd2=0,dfd3=0,dfd4=0,dfd5=0,dfd7=0,dmdh=0,dmnd_run=0,en_demand=0

Any obvious things I need to do next? (I was going to post this in the issues section, but suspect I was either not submitting it correctly or double-submitting it. Feel free to move/delete this post if you want).

**Comment by tiekenz** [comment]
![faikin screenshot](https://github.com/revk/ESP32-Faikin/assets/17054368/c54eaff5-75cc-4af6-bd76-03a1edab6635)
![faikin module](https://github.com/revk/ESP32-Faikin/assets/17054368/c51585f3-b57e-4280-97f9-bb417d57522f)
![daughterboard](https://github.com/revk/ESP32-Faikin/assets/17054368/40b297d8-8e89-4f16-a425-5428795c8b86)

**Reply by revk** [reply]
There may be others that can confirm the wiring, but it looks right to me.

I am assuming it should work the same as any other S21.

One test you can do...

1. Loopback, ie connect pin 2 to pin 3 on Faikin module

2. Connect the power (which are pin 5 and 4) the Faikin module should light up.

This should be detected by the latest firmware as a loopback and blink red, then green, then blue, repeatedly.

If that happened it confirms the Faikin module is not actually broken.
